components/automate-chef-io/content/docs/restore.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "Restore"
+description = "Procedures for Restoring Chef Automate Data"
+date = 2018-03-26T15:27:52-07:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "get_started"
+    weight = 80
++++
+
+Restore Chef Automate from a [filesystem backup]({{< ref "restore.md#restore-from-a-filesystem-backup" >}}) or an [Amazon S3 bucket backup]({{< ref "restore.md#restore-from-an-s3-backup" >}}).
+
+## Prerequisites
+
+1. On the restore host, download and unzip the Chef Automate command-line tool:
+
+   ```shell
+        curl https://packages.chef.io/files/current/latest/chef-automate-cli/chef-automate_linux_amd64.zip | gunzip - > chef-automate && chmod +x chef-automate
+    ```
+
+1. **Filesystem backups** require access for Chef Automate to a backup directory in the [configured location]({{< ref
+"backup.md#backup-to-a-filesystem" >}}).
+Ensure access for the backup type used:
+
+     1. To restore a network-attached filesystem backup, mount the shared backup directory to the same mount point configured at the time of the backup.
+     1. To restore a backup directory that is not a network-attached filesystem, copy the backup directory to the [configured location]({{< ref "backup.md#backup-to-a-filesystem" >}}) at the time of the backup.
+     1. For restoring a single-file backup archive, copy your archive to the restore host and extract it to the [configured backup directory]({{< ref "backup.md#backup-to-a-filesystem" >}}).
```

Comment:
@yzl Would [the header from backup.md, line 50](https://github.com/chef/automate/pull/3937/files#diff-5d531e3aed08d5efb3442a84b4e0dd56R55) be a better link to use here  rather than "backup to a file system"?

Reply:
I don't think so. At this point in the restore process, the users already has a single-file archive in hand that they want to use for the restore. What they need to know is that they have to put the contents of the archive in the directory they've told Automate to look in for backup data.

Reply:
Ah, okay. Thank you for clarifying! I think I will proceed with the plan of action for links as laid out in the comment above.